//package com.sincetimes.redwar.game.comnunicate.protocol;

message RseUpdateSocialItem
{
	optional int32 ret = 1;
}

